docs/installation.md

```diff
@@ -64,6 +64,14 @@ $ eval $(docker-machine env default)
     # in ~/<your-path-to-cloned-repo>/amundsensearchlibrary
     $ docker build -f public.Dockerfile -t amundsen-search:latest .
     ```
+  * Or pull docker image directly from dockerhub
+    ```bash
+    $ docker pull docker pull photoft/amundsen-frontend
```

Comment:
@feng-tao I believe you need to remove the one `docker pull` from each statement. It is duplicated